datalake-api/src/main/java/com/sequenceiq/sdx/api/model/ModelDescriptions.java

```diff
@@ -10,6 +10,8 @@ public class ModelDescriptions {
 
     public static final String CLOSE_CONNECTIONS = "The conditional parameter for whether connections to the database will be closed during backup or not.";
 
+    public static final String SKIP_DATABASE_NAMES = "The names of the Database to not be backed up, or blank for all";
```

Comment:
I would say "The names of the Databases" since it seems like it should be plural.